content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
 
-We recommend following [these instructions](https://babeljs.io/docs/editors/) to configure syntax highlighting for your editor.
+हम आपके संपादक के लिए वाक्य रचना हाइलाइटिंग को कॉन्फ़िगर करने के लिए [इन निर्देशों](https://babeljs.io/docs/editors/) का पालन करने की सलाह देते हैं।
 
 </details>
 
-### Help, I'm Stuck! {#help-im-stuck}
+### मदद, मैं अटक गया हूँ! {#help-im-stuck}
 
-If you get stuck, check out the [community support resources](/community/support.html). In particular, [Reactiflux Chat](https://discord.gg/0ZcbPKXt5bZjGY5n) is a great way to get help quickly. If you don't receive an answer, or if you remain stuck, please file an issue, and we'll help you out.
+यदि आप अटक जाते हैं, तो [सामुदायिक सहायता संसाधनों](/community/support.html) की जांच करें। विशेष रूप से, [Reactiflux चैट](https://discord.gg/0ZcbPKXt5bZjGY5n) जल्दी से सहायता प्राप्त करने का एक शानदार तरीका है। यदि आपको कोई उत्तर नहीं मिलता है, या यदि आप अटके रहते हैं, तो कृपया एक समस्या दर्ज करें, और हम आपकी मदद करेंगे।
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-Now that you're set up, let's get an overview of React!
+अब जब आप सेट हो गए हैं, तो React का ओवरव्यू करें!
 
-### What Is React? {#what-is-react}
+### React क्या है? {#what-is-react}
 
-React is a declarative, efficient, and flexible JavaScript library for building user interfaces. It lets you compose complex UIs from small and isolated pieces of code called "components".
+उपयोगकर्ता इंटरफेस के निर्माण के लिए React एक घोषणापत्र, कुशल और लचीली जावास्क्रिप्ट लाइब्रेरी है। यह आपको "कौम्पोनॅन्ट्स" नामक कोड के छोटे और पृथक टुकड़ों से जटिल यूआई की रचना करने देता है।
 
-React has a few different kinds of components, but we'll start with `React.Component` subclasses:
+React के कुछ अलग-अलग प्रकार के कौम्पोनॅन्ट होते हैं, लेकिन हम `React.Component` उपवर्गों के साथ शुरुआत करेंगे:
 
 ```javascript
 class ShoppingList extends React.Component {
   render() {
     return (
       <div className="shopping-list">
-        <h1>Shopping List for {this.props.name}</h1>
+        <h1>{this.props.name} के लिए खरीदारी की सूची</h1>
         <ul>
-          <li>Instagram</li>
-          <li>WhatsApp</li>
-          <li>Oculus</li>
+          <li>इंस्टाग्राम</li>
+          <li>व्हाट्सैप्</li>
+          <li>ओकुलस</li>
         </ul>
       </div>
     );
   }
 }
 
-// Example usage: <ShoppingList name="Mark" />
+// उपयोग उदाहरण: <ShoppingList name="Mark" />
 ```
 
-We'll get to the funny XML-like tags soon. We use components to tell React what we want to see on the screen. When our data changes, React will efficiently update and re-render our components.
+हम जल्द ही मज़ेदार एक्सएमएल-जैसे टैग प्राप्त करेंगे. हम स्क्रीन पर जो देखना चाहते हैं उसे React करने के लिए हम कौम्पोनॅन्ट्स का उपयोग करते हैं। जब हमारा डेटा बदलता है, तो React कुशलतापूर्वक हमारे कौम्पोनॅन्ट्स को अद्यतन और प्रस्तुत करेगा।
```

Comment:
मज़ेदार एक्सएमएल-जैसे -> gives the sentence a different meaning

Reply:
How about
`हमें जल्दी ही हास्यजनक XML टैग्स मिलेंगे` ?

Reply:
अजीब instead of  हास्यजनक ?